Automattic/wp-calypso
# Assembler: Refactor by prefixing Pages components and hooks with Pattern

Related to #83450

## Proposed Changes

This PR proposes to prefix Pages components and hooks with the word Pattern, to denote that pages are also patterns. This PR also propose to reuse the same hook `useCategoriesOrder` to sort both section and page list.

## Testing Instructions



* Head to the Assembler with the flag `&flags=pattern-assembler/add-pages`.
* Go through the Patterns, and Styles screen until you arrive at the Pages screen.
* Ensure that the Add Pages screen works as before.
* Ensure that the Section category list is ordered as before. 

## Pre-merge Checklist
